src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -406,6 +406,12 @@
 type: command
 short-summary: Queues a quick run providing streamed logs for an Azure Container Registry.
 examples:
+  - name: Queue a contextless run.
+    text: >
+        az acr run -r MyRegistry --cmd bash /dev/null
+  - name: Queue a run from standard input. Either 'Ctrl + z' or 'Ctrl + d' terminates the input stream.
+    text: >
+        az acr run -r MyRegistry -f - .
```

Comment:
How does user know `-f` take a value of `-`? We probably need to update help message for the argument `-f` here. // cc @northtyphoon

Reply:
Examples might not be conspicuous enough to catch the attention. Also added in the argument list.